extension/docs/CreditCardIntegration.md

```diff
@@ -0,0 +1,67 @@
+## Credit card payment
+
+<!-- START doctoc generated TOC please keep comment here to allow auto update -->
+<!-- DON'T EDIT THIS SECTION, INSTEAD RE-RUN doctoc TO UPDATE -->
+**Contents**
```

Comment:
I would not use a header for this.

Reply:
This is autogenerated. I think this format is good.